Dockerfile.development

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.11-stretch
+
+ENV _RUBY_VERSION 2.5.3
+ENV _RBENV_VERSION  v1.1.1
+ENV _RBUILD_VERSION v20190130
+
+# Ruby and mruby deps
+RUN apt update && \
+    apt -y upgrade && \
+    apt -y install bzip2 autoconf bison build-essential libssl-dev libyaml-dev \
+                   libreadline-dev zlib1g-dev libncurses5-dev libffi-dev \
+                   libgdbm3 libgdbm-dev
+
+RUN git clone https://github.com/sstephenson/rbenv.git --branch $_RBENV_VERSION ~/.rbenv
+RUN cd ~/.rbenv && src/configure && make -C src
+ENV PATH ~/.rbenv/bin:~/.rbenv/shims:$PATH
+RUN echo 'eval "$(rbenv init -)"' >> /etc/profile.d/rbenv.sh # or /etc/profile
+RUN echo 'eval "$(rbenv init -)"' >>  ~/.bashrc
+
+RUN git clone https://github.com/sstephenson/ruby-build.git --branch $_RBUILD_VERSION ~/.rbenv/plugins/ruby-build
+RUN ~/.rbenv/plugins/ruby-build/install.sh
+RUN eval ~/.rbenv/bin/rbenv install $_RUBY_VERSION && ~/.rbenv/bin/rbenv global $_RUBY_VERSION
+
+ENV APP ${GOPATH}/src/github.com/anycable/anycable-go
+COPY . ${APP}
```

Comment:
Do we need to copy all the files for development if we mount the volume with the current dir?

Reply:
We can omit it if assume to run ‘make prepare’ manually.

Reply:
> to run ‘make prepare’ manually.

seems reasonable

Reply:
So, you want it to be run manually or not? The idea was to build an image and be ready for development without any additional steps. Do you see any tradeoffs with that approach? `make prepare` installs `anyt` gem and it would be cool to have it inside the image without the necessity to create persistent volumes.

Reply:
> The idea was to build an image and be ready for development without any additional steps

tbh, I don't see any reason for doing this) Why do you want to develop it within a Docker? To avoid dealing with mruby locally?
What is the difference between supporting deps within a Docker image and on the host machine?